tests/addons.test.js

```diff
@@ -1,16 +1,8 @@
-const path = require('path')
 const test = require('ava')
 const stripAnsi = require('strip-ansi')
 const cliPath = require('./utils/cliPath')
-const exec = require('./utils/exec')
```

Comment:
`./utils/exec` is a promisified version of node `exec`.
We were actually spawning commands in 3 different ways in the tests.
1. Using `./utils/exec` (addons tests)
2. Using a promisified `exec` duplicating `./utils/exec` logic (dev command tests)
3. Using `execa` (build command tests).

I aligned the places I could find to use `execa` and removed `./utils/exec`

Reply:
YES! :tada: 

Child process spawning is complex :P